frontend/src/components/common/GeneralDeleteModal.tsx

```diff
@@ -11,26 +12,25 @@ import {
   ModalOverlay,
   Text,
 } from "@chakra-ui/react";
-import React from "react";
 
-interface GeneralDeleteCamperModalProps {
+interface GeneralDeleteModalProps {
   title: string;
   bodyText: string;
   bodyNote?: string;
   buttonLabel: string;
   isOpen: boolean;
   onClose: () => void;
-  onRemove: () => void;
+  onDelete: () => void;
 }
-const RemoveCamperModal = ({
+const GeneralDeleteModal = ({
```

Comment:
Hmm not a strong opinion, but having this component in `/common/` implies the "General" part to me. Aka - thoughts on just calling this `DeleteModal`?

Reply:
yea fs omw